.github/scripts/test-onnx-export.sh

```diff
@@ -205,7 +322,7 @@ GIT_LFS_SKIP_SMUDGE=1 git clone $repo_url
 repo=$(basename $repo_url)
 
 pushd $repo
-git lfs pull --include "data/lang_bpe_500/bpe.model"
+git lfs pull --include "data/lang_bpe_500/tokens.txt"
```

Comment:
```suggestion
```

`tokens.txt` in
https://huggingface.co/csukuangfj/icefall-asr-librispeech-pruned-transducer-stateless7-2022-11-11/tree/main/data/lang_bpe_500
is not tracked by `git-lfs`.